### DISCUSSION TITLE
Unknown error while trying to spawn actor

### DISCUSSION THREAD

**Discussion by ShaharShakiUnreal** [carla-simulator/carla#7702] [Q&A]
Hi I'm trying to spawn an object I imported into Carla. I followed this guide:
https://carla.readthedocs.io/en/latest/tuto_A_add_props/#ingestion-in-a-build-from-source
And indeed the object is exposed in blueprint_library. But when I try to spawn it I get the following error:
'Unknown error while trying to spawn actor'
Does anyone know why this is happening?

This is my code:
![image](https://github.com/carla-simulator/carla/assets/149073174/6a0d9c12-8629-4dd6-9c92-aad167632f8a)

the blue arrow is where I get the exception.

Needless to say, I have no problem spawning the other objects.

Thanks.

**Comment by xavisolesoft** [comment]
Could you share the error message and indicate what version of CARLA are you using, please?
